src/components/Navigation/tests/Navigation.test.tsx

```diff
@@ -75,31 +73,4 @@ describe('<Navigation />', () => {
       expect(navigation.contains(contextControl)).toBe(true);
     });
   });
-
-  describe('newDesignLanguage', () => {
-    it('renders an image if newDesignLanguage is true and a logo from the theme provider is present', () => {
```

Comment:
This test looks like it might still be valuable. Just reword it so it makes no mention of the new design language